src/coreclr/jit/jitconfigvalues.h

```diff
@@ -116,7 +116,8 @@ CONFIG_INTEGER(JitNoHoist, W("JitNoHoist"), 0)
 CONFIG_INTEGER(JitNoInline, W("JitNoInline"), 0)                 // Disables inlining of all methods
 CONFIG_INTEGER(JitNoMemoryBarriers, W("JitNoMemoryBarriers"), 0) // If 1, don't generate memory barriers
 CONFIG_INTEGER(JitNoRegLoc, W("JitNoRegLoc"), 0)
-CONFIG_INTEGER(JitNoStructPromotion, W("JitNoStructPromotion"), 0) // Disables struct promotion in Jit32
+CONFIG_INTEGER(JitNoStructPromotion, W("JitNoStructPromotion"), 0) // Disables struct promotion &1 - for all, &2 - for
```

Comment:
nit: What does `&1` and `&2` mean?

Reply:
I do not know how to describe it, it means 
```
if ((JitNoStructPromotion & 1) != 0) fgNoStructPromotion = true
if ((JitNoStructPromotion & 2) != 0) fgNoStructParamPromotion= true
```

we use a similar "hash" approach for JitStress and JitMinOpts but they don't have such comments.

Reply:
I see. Why not just pass `JitNoStructPromotion=1`, etc. and then check for `if (JitNoStructPromotion  == 1) fgNoStructPromotion = true` and so forth?